src/postGithubIssue.ts

```diff
@@ -72,7 +72,10 @@ ${Object.keys(statusCounts).sort().map(status => `| ${status} | ${statusCounts[s
 `;
 
 const resultPaths = pu.glob(resultDirPath, `**/*.${resultFileNameSuffix}`).sort((a, b) => path.basename(a).localeCompare(path.basename(b)));
-const outputs = resultPaths.map(p => fs.readFileSync(p, { encoding: "utf-8" }).replace(new RegExp(artifactFolderUrlPlaceholder, "g"), artifactsUri));
+const outputs = resultPaths.map(p =>
+    fs.readFileSync(p, { encoding: "utf-8" })
+        .replace(new RegExp(artifactFolderUrlPlaceholder, "g"), artifactsUri)
+        .replace(new RegExp(getArtifactsApiUrlPlaceholder, "g"), getArtifactsApi));
```

Comment:
These could just be replaceAll (I believe we run in node 18+ only)

Reply:
Done.